framework/src/state_machine/state_machine.ts

```diff
@@ -205,9 +193,21 @@ export class StateMachine {
 					this._logger.debug({ moduleName: mod.name }, 'Executing afterCommandExecute');
 					await mod.afterCommandExecute(transactionContext);
 					this._logger.debug({ moduleName: mod.name }, 'Executed afterCommandExecute');
+					ctx.eventQueue.unsafeAdd(
```

Comment:
standard event should be added based on the result of the execution.
if transaction is invalid, the event should not be added